api/python/indigo/indigo/indigo_object.py

```diff
@@ -26,6 +27,14 @@
     from .indigo import Indigo
 
 
+class IndigoObjectType(Enum):
```

Comment:
indigoObject already has a type inside C++ code. Please remove it from python

Reply:
Decided not to propagate indigo object class for now